<ui version="4.0" >
 <class>QStrokeSegmentationExtensionBase</class>
 <widget class="QWidget" name="QStrokeSegmentationExtensionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QSplitter" name="m_verticalSplitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="m_horizontalSplitter" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget_2" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="m_slicingToolButton" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_windowLevelToolButton" >
            <property name="text" >
             <string>window level</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_zoomToolButton" >
            <property name="text" >
             <string>zoom</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_moveToolButton" >
            <property name="text" >
             <string>move</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_rotateClockWiseToolButton" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_editorToolButton" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_voxelInformationToolButton" >
            <property name="layoutDirection" >
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text" >
             <string>voxel information</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="m_seedToolButton" >
            <property name="text" >
             <string>...</string>
            </property>
            <property name="icon" >
             <iconset/>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="autoRaise" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="m_viewThresholdButton" >
          <property name="layoutDirection" >
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text" >
           <string>View thresholds</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Lower value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_lowerValueSlider" >
            <property name="maximum" >
             <number>255</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_lowerValueSpinBox" >
            <property name="maximum" >
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_2" >
            <property name="text" >
             <string>Upper value:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_upperValueSlider" >
            <property name="maximum" >
             <number>255</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_upperValueSpinBox" >
            <property name="maximum" >
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="m_opacitiyLabel" >
            <property name="text" >
             <string>Opacity:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_opacitySlider" >
            <property name="maximumSize" >
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="value" >
             <number>50</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="m_seedLabel" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>Seed:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_seedXLineEdit" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_seedYLineEdit" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_seedZLineEdit" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="m_applyMethodButton" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection" >
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text" >
             <string>Apply</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="m_resultsLabel" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Volume (in mm3):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_resultsLineEdit" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>4</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_updateVolumeButton" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Update</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupEditor" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title" >
             <string>Editor</string>
            </property>
            <widget class="QWidget" name="layoutWidget_4" >
             <property name="geometry" >
              <rect>
               <x>20</x>
               <y>30</y>
               <width>321</width>
               <height>30</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QPushButton" name="m_eraseButton" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>1</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="icon" >
                 <iconset/>
                </property>
                <property name="flat" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_paintButton" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="sizePolicy" >
                 <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>1</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="icon" >
                 <iconset/>
                </property>
                <property name="checkable" >
                 <bool>false</bool>
                </property>
                <property name="checked" >
                 <bool>false</bool>
                </property>
                <property name="flat" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_eraseSliceButton" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="text" >
                 <string/>
                </property>
                <property name="icon" >
                 <iconset/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>Size</string>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="m_editorSize" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="minimum" >
                 <number>0</number>
                </property>
                <property name="value" >
                 <number>5</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="udg::Q2DViewer" native="1" name="m_2DView" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>100000</width>
           <height>100000</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="m_sliceLabel" >
           <property name="text" >
            <string>Slice:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="m_sliceViewSlider" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_sliceSpinBox" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::Q2DViewer</class>
   <extends>QWidget</extends>
   <header>q2dviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_upperValueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_upperValueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>236</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel" >
     <x>309</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_upperValueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_upperValueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>309</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lowerValueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_lowerValueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>240</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel" >
     <x>427</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lowerValueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_lowerValueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>427</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel" >
     <x>240</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sliceSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_sliceViewSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>501</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel" >
     <x>425</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sliceViewSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m_sliceSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>425</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel" >
     <x>501</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
